src/pocketdb/web/PocketContentRpc.cpp

```diff
@@ -1596,4 +1596,64 @@ namespace PocketWeb::PocketWebRpc
     },
         };
     }
+
+    RPCHelpMan GetsubsciptionsGroupedByAuthors()
+    {
+        return RPCHelpMan{"GetsubsciptionsGroupedByAuthors",
+                          "\n\n", // TODO (rpc)
+                          {
+                                  {"address", RPCArg::Type::STR, RPCArg::Optional::NO, "" /* TODO (rpc): arg description*/},
+                                  {"addressPagination", RPCArg::Type::STR, RPCArg::Optional::OMITTED_NAMED_ARG, "" /* TODO (rpc): arg description*/},
+                                  {"nHeight", RPCArg::Type::NUM, RPCArg::Optional::OMITTED_NAMED_ARG, "" /*TODO (rpc): arg description*/},
+                                  {"countOutOfUsers", RPCArg::Type::NUM, RPCArg::Optional::OMITTED_NAMED_ARG, "" /* TODO (rpc): arg description*/},
+                                  {"countOutOfcontents", RPCArg::Type::STR, RPCArg::Optional::OMITTED_NAMED_ARG, "" /* TODO (rpc): arg description*/}
+                          },
+                          {
+                                  // TODO (rpc): provide return description
+                          },
+                          RPCExamples{
+                                  // TODO (rpc): better examples
+                                  HelpExampleCli("getsubsciptionsgroupedbyauthors", "...") +
+                                  HelpExampleRpc("getsubsciptionsgroupedbyauthors", "...")
+                          },
+                          [&](const RPCHelpMan& self, const JSONRPCRequest& request) -> UniValue
+                          {
+                              string address = "";
+                              string addressPagination = "";
+                              int nHeight = ChainActive().Height();
+                              int countOutOfUsers = 10;
+                              int countOutOfcontents = 10;
+
+                              UniValue result(UniValue::VOBJ);
+
+                              if (request.params.empty())
+                                  return result;
+
+                              if (request.params[0].isStr())
+                                  address = request.params[0].get_str();
+
+                              if (request.params.size() > 1 && request.params[1].isStr())
+                                  addressPagination = request.params[1].get_str();
+
+                              if (request.params.size() > 2 && request.params[2].isNum() && request.params[2].get_int() > 0)
+                                  nHeight = request.params[2].get_int();
+
+                              if (request.params.size() > 3 && request.params[3].isNum())
+                                  countOutOfUsers = std::max(request.params[3].get_int(), 100);
```

Comment:
It is supposed to call `std::min`, isn't it? Otherwise the default values specified above will be always overridden.